services/userServices.js

```diff
@@ -0,0 +1,29 @@
+const jwt = require('jsonwebtoken');
+require('dotenv').config();
+const userDao = require('../models/userDao');
+const { axiosUser } = require('../util/axios');
+
+const loginService = async (kakaoToken) => {
+  try {
+    const user = await axiosUser(kakaoToken);
```

Comment:
해당 부분은 user 보다는 kakao 정보가 담기는 변수이므로 네이밍을 바꾸는 것이 좋을 것 같습니다.

```suggestion
    const kakaoUserInfo = await axiosUser(kakaoToken);
```